src/generated/resources/data/gtceu/tags/items/sensors.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_sensor",
+    "gtceu:mv_sensor",
+    "gtceu:hv_sensor",
+    "gtceu:ev_sensor",
+    "gtceu:iv_sensor",
+    "gtceu:luv_sensor",
+    "gtceu:zpm_sensor",
+    "gtceu:uv_sensor",
+    "gtceu:uhv_sensor",
+    "gtceu:uev_sensor",
+    "gtceu:uiv_sensor",
+    "gtceu:uxv_sensor",
+    "gtceu:opv_sensor"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.